src/networking/Packets/MsgAddPlayer.php

```diff
@@ -38,10 +38,19 @@ class MsgAddPlayer extends GamePacket
 
     /**
      * @since 1.0.0
+     * @deprecated since 1.1, use getPlayerId instead
```

Comment:
```suggestion
     * @deprecated 1.1 use `MsgAddPlayer::getPlayerId()` instead
```

Take a look at the [syntax for `@deprecated`](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#54-deprecated)